src/components/generic/confirm-dialog.tsx

```diff
@@ -42,38 +42,39 @@ export default class ConfirmDialog extends React.Component<Props, State> {
 
     const dialogText = confirmDialogData.deletePossible ? confirmDialogData.text : confirmDialogData.contentTitle;
 
-    return
+    return (
       <>
-      <Dialog
```

Comment:
Should this be converted to a functional component?

Reply:
Discussed with Ville, initially decided not to do this, not it is being done. Expect new commit from Ville soon